src/kernel/src/memory/mod.rs

```diff
@@ -513,6 +513,7 @@ bitflags! {
     /// Flags to tell what access is possible for the virtual page.
     #[derive(Debug, Clone, Copy, PartialEq)]
     pub struct Protections: u32 {
+        const NONE = 0x00000000;
```

Comment:
Bit flag with zero value cannot be tested to see if it is present in the set.

Reply:
Eh, was just following how FreeBSD uses 0x00 as None, plus I don't see a way to give the Linux page Prot_None without this.

Reply:
You can use `Protections::empty()` to achieve this.